smart-contracts/tasks/origins/add.ts

```diff
@@ -287,7 +282,30 @@ task('pool:remove-origin', 'Removes an origin from a pool')
         process.exit()
       }
 
-      const tx = await pool.disableOrigin(l2ChainId, bridgeAddress)
-      console.log(`✅ Transaction sent! ${tx.hash}`)
+      if (origin.curator === userAddress) {
+        const tx = await pool.disableOrigin(l2ChainId, bridgeAddress)
+        console.log(`✅ Transaction sent! ${tx.hash}`)
+        return
+      }
+
+      // Else, let's check that maybe the curator is set?
+      const defaultCurator =
+        (await getOriginCuratorForNetwork(Number(chainId))) || userAddress
+      if (origin.curator !== defaultCurator) {
+        throw Error(
+          `You are not the curator of this origin. Please contact the curator (${origin.curator}) to disable it.`
```

Comment:
here that error message is unclear. Should be sth like the "curator is not a forwarder" ?